### DISCUSSION TITLE
What current status of support for the Nim programming language?

### DISCUSSION THREAD

**Discussion by DAMisener** [universal-ctags/ctags#4055] [Q&A]
`ctags --list-languages` would indicate that **Nim** not a supported language.

I see several **closed** messages  in the *issue tracker* which reference work regarding Nim support.

- [add nim language parser and fix makefile](https://github.com/universal-ctags/ctags/pull/401)
- [add nim language parser](https://github.com/universal-ctags/ctags/pull/396)
- [Revert "add nim language parser"](https://github.com/universal-ctags/ctags/pull/395)
- [add nim language parser](https://github.com/universal-ctags/ctags/pull/393)

These have all been **closed**.  

Has Nim support been abandoned?

BTW: *I am aware of the existence of tags*

**Comment by masatake** [comment]
See #401. None is working on the parser.

This is not an issue. So I converted to a "discussion" item.
